test/install_test.go

```diff
@@ -19,11 +19,22 @@ import (
 ///   TEST SETUP   ///
 //////////////////////
 
-var TestHelper *testutil.TestHelper
+var (
+	TestHelper *testutil.TestHelper
+
+	// fetchLogsEvents is a switch to enable calling Testlogs and TestEvents
+	// when a test fails
+	fetchLogsEvents = false
+)
 
 func TestMain(m *testing.M) {
 	TestHelper = testutil.NewTestHelper()
-	os.Exit(m.Run())
+	code := m.Run()
+	// if the test failed, show logs and events that can potentially explain cause
+	if code != 0 && fetchLogsEvents {
```

Comment:
I think the logic is clearer here if we rename the fetchLogsEvents variable to controlPlaneInstalled.  This makes it obvious that we only want to fetch logs and events if the control plane has already been installed.